### DISCUSSION TITLE
Hold abilities for burst?

### DISCUSSION THREAD

**Discussion by SaroAfk** [Hekili/hekili#833] [General]
Awesome AddOn, loving it. I'm finding the logic behind the ability queues to be very useful.

I am wanting to hold on to 2 or 3 abilities so that I can use them together in a burst window. However, the queue logic starts getting really wonky when I just leave them active and off cooldown in the Primary position. Things start queueing up expecting to have that GCD time frame to come off cooldown.

I am wondering if there is a macro command or something that can remove abilities from the queue and then add them back in when I want to burst?

**Comment by Hekili** [comment]
See Toggles.
